Source/NSButton.m

```diff
@@ -543,4 +547,43 @@ - (NSSound *) sound
   return [_cell sound];
 }
 
+- (void) mouseDown: (NSEvent *)event
+{
+  [super mouseDown: event];
+
+  if ([[self superview] isKindOfClass: [NSMatrix class]] == NO)
+  {
+    NSImage *i = [[self cell] image];
```

Comment:
Maybe we should extract the decision whether an NSButton is a radio button into a separate helper method. That would make it easier to change the actual implementation when we find out that different criteria are needed.

Reply:
Done.